src/button.js

```diff
@@ -0,0 +1,13 @@
+export default (parentId, name, onclick) => {
+  const button = document.createElement('button')
+  button.textContent = name
+  button.onclick = onclick
+  document.getElementById(parentId).appendChild(button)
+
+  return () => {
+    button.parentNode.removeChild(button)
```

Comment:
the correct element will be removed.

> The removeChild() method removes a specified child node of the specified element
more info: https://developer.mozilla.org/en-US/docs/Web/API/Node/removeChild